app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PetApplicationsController < ApplicationController
+
+  def create
+    applicant = Application.find(params[:id])
+    pet_application = PetApplication.create!({
+    pet_id: params[:pet_id],
+    application_id: params[:application_id],
```

Comment:
This was cool to see that we can reutilize the [:id] and pass that through the application_id to assign it to that new PetApplication object.